src/proxy/PNCounter.ts

```diff
@@ -29,21 +29,21 @@ import {DistributedObject} from '../core/DistributedObject';
  * communicate will eventually have the same data.
  *
  * The invocation is remote. This may lead to indeterminate state -
- * the update may be applied but the response has not been received.
+ * the update may be applied, but the response has not been received.
  * In this case, the caller will be notified with a `TargetDisconnectedError`
  *
  * The read and write methods provide monotonic read and RYW (read-your-write)
  * guarantees. These guarantees are session guarantees which means that if
  * no replica with the previously observed state is reachable, the session
- * guarantees are lost and the method invocation will throw a
+ * guarantees are lost, and the method invocation will throw a
  * `ConsistencyLostError`. This does not mean
- * that an update is lost. All of the updates are part of some replica and
+ * that an update is lost. All the updates are part of some replica and
```

Comment:
Reference for use of "all of" and "all": https://jakubmarian.com/is-it-all-the-or-all-of-the-in-english/